internal/commands/build.go

```diff
@@ -261,6 +263,7 @@ func buildCommandFlags(cmd *cobra.Command, buildFlags *BuildFlags, cfg config.Co
 	cmd.Flags().StringVar(&buildFlags.DateTime, "creation-time", "", "Desired create time in the output image config. Accepted values are Unix timestamps (e.g., '1641013200'), or 'now'. Platform API version must be at least 0.9 to use this feature.")
 	cmd.Flags().StringVarP(&buildFlags.DescriptorPath, "descriptor", "d", "", "Path to the project descriptor file")
 	cmd.Flags().StringVarP(&buildFlags.DefaultProcessType, "default-process", "D", "", `Set the default process type. (default "web")`)
+	cmd.Flags().BoolVar(&buildFlags.DisableSystemBuilpacks, "disable-system-buildpacks", false, "Disable System Buildpacks")
```

Comment:
Do we need/want this flag on `builder create`?

Reply:
It could be useful if a Buildpack Author wants to create a builder version without any System Buildpack on it?, I don't know, let's say I am maintaining a builder, I added my System Buildpacks to `builder.toml` but I want to also publish a version without them, I can use the `--disable-system-buildpacks` flag maybe to do it without having to touch anything else. Does it make sense?